hw03_wange2.py

```diff
@@ -0,0 +1,73 @@
+# Name: Eun Sung Wang
+# Username: wange2
+# Google Doc: https://docs.google.com/document/d/1jl7uXjys2F0Tu1IFPvjZcEspKPECxvctfCDyoPJD6h8/edit?usp=sharing
+
+import turtle
+
+def drawBase(eunsung):            #Function to draw the base of the house
```

Comment:
All functions should have docstrings, formatted correctly (on the line after the definition, using a triple-quote string).

Reply:
I have added them.